homeassistant/components/cover/knx.py

```diff
@@ -34,6 +39,9 @@
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_SETPOSITION_ADDRESS): cv.string,
     vol.Optional(CONF_INVERT_POSITION, default=False): cv.boolean,
+    vol.Optional(CONF_GETANGLE_ADDRESS): cv.string,
+    vol.Optional(CONF_SETANGLE_ADDRESS): cv.string,
```

Comment:
In this case, I don't understand how the code should look like.
Can you give me a full example for this?

While it would be possible to define regexps for the addresses, these would become quite ugly (there are multiple address formats).